python/paddle/v2/fluid/tests/test_layers.py

```diff
@@ -130,13 +130,19 @@ def test_word_embedding(self):
     def test_linear_chain_crf(self):
         program = Program()
         with program_guard(program, startup_program=Program()):
+            label_dict_len = 10
             images = layers.data(name='pixel', shape=[784], dtype='float32')
```

Comment:
I personally suggest using NLP related names rather than names for images here, but it has little effect.